aikau/src/main/resources/alfresco/css/less/defaults.less

```diff
@@ -1,3 +1,17 @@
+// Main palette
+@primaryBG: #fff;
+@primaryFG: #333;
+@headerBG: #0082c8;
```

Comment:
This fine for now, however - with AKU-286 we might need to change these to be black to match the current header colour.